modules/world/tests/label_test.cc

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2020 fortiss GmbH
```

Comment:
should we split up the world tests as well in subfolders?

Reply:
Do you mean like `modules/world/evaluation/labels/tests/label_test.cc` ?

Reply:
for example, or just split the test folder within world/tests/ as the number of tests is steadily increasing.